pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller.go

```diff
@@ -65,6 +67,8 @@ type NSXServiceAccountReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.0/pkg/reconcile
 func (r *NSXServiceAccountReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	// Use once.Do to ensure gc is called only once
+	once.Do(func() { go r.GarbageCollector(make(chan bool), servicecommon.GCInterval) })
```

Comment:
GC won't work until at least one resource is reconciled. Is it acceptable?

Reply:
Yes